samples/helloworld/README.md

```diff
@@ -40,6 +40,7 @@ arguments:
 
 Argument | Default | Description
 -------- | ------- | -----------
+`-h`,`--help` | ` ` | Prints usage information.
```

Comment:
making the default a single space char doesn't display so nice on github. Can you change it to no default like this?
```suggestion
`-h`,`--help` | | Prints usage information.
```

Reply:
Updated